sdk/identity/identity/review/identity.api.md

```diff
@@ -34,10 +34,11 @@ export const AuthenticationErrorName = "AuthenticationError";
 // @public
 export interface AuthenticationRecord {
     authority: string;
+    clientId: string;
     homeAccountId: string;
-    serialize: () => string;
     tenantId: string;
     username: string;
+    version: SupportedAuthenticationRecordVersions;
```

Comment:
I been wondering if these should be simple strings? Is this client -> service or service -> client?

Reply:
this is client-side only. The idea is that over time we will change this and we'll have a way to say "we only support these versions". For now we only support v `1.0`

Reply:
Personally, I vote this to be a string - especially since it's just serialized / deserialized, I dont know what the benefit of making this typed would be and adding it to the public API. What do you think?

Reply:
This property has been removed from the public API after Scott's feedback here: https://github.com/Azure/azure-sdk-for-js/pull/14630#discussion_r606487554